scripts/build_utils.sh

```diff
@@ -1398,16 +1398,6 @@ setup_rpm_build_deps() {
 
     sed -e 's/@//g' < ceph.spec.in > $DIR/ceph.spec
 
-    # enable more build depends required by build flavor(jaeger, crimson)
-    case "${FLAVOR}" in
-    crimson)
-        sed -i -e 's/%bcond_with seastar/%bcond_without seastar/g' $DIR/ceph.spec
-        ;;
-    jaeger)
-	sed -i -e 's/%bcond_with jaeger/%bcond_without jaeger/g' $DIR/ceph.spec
-        ;;
```

Comment:
this change should be squashed into the change of "build_utils: export jaeger build args if FLAVOR == jaeger".

Reply:
and please note, `yum-builddep` does not consume `--with jaeger/crimson`. so i think we probably have to live with this.

Reply:
:( added to https://github.com/ceph/ceph-build/pull/1862/commits/13a4afe33e55147c08af86739e650e0e9266689e